components/compass-runtime-agent/internal/k8sconsts/nameresolver.go

```diff
@@ -0,0 +1,62 @@
+package k8sconsts
```

Comment:
We should consider moving this stuff to some common library as we have the same code in Application Registry.

Reply:
We should do it at once with name normalization